docs/src/pages/docs/crocks/Async.md

```diff
@@ -1492,6 +1502,16 @@ Resolved([ 'cat', 'bat', 'imp' ])
   .chain(findFirstValid)
   .fork(log('rej'), log('res'))
 //=> rej: "Nothing Found"
+
+Resolved(First.empty())
+  .chain(firstToAsync('Left'))
+  .fork(log('rej'), log('res'))
+//=> rej: "Left"
+
+Resolved(First(42))
+  .chain(firstToAsync('Error'))
```

Comment:
Can we use Left her instead of Error, so we maintain the same logic btw the examples and it's easier to compare

Reply:
Yeah not sure why i changed it lol